### DISCUSSION TITLE
Welcome to playactor Discussions!

### DISCUSSION THREAD

**Discussion by dhleong** [dhleong/playactor#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by donkeykong74** [comment]
he siri, turn on playstation.. love it! thanks so much, also crushing the random turn-on bug..
